services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -0,0 +1,109 @@
+package fi.hel.haitaton.hanke.hakemus
+
+import assertk.Assert
+import assertk.assertThat
+import assertk.assertions.isEmpty
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotNull
+import assertk.assertions.prop
+import assertk.assertions.single
+import fi.hel.haitaton.hanke.DatabaseTest
+import fi.hel.haitaton.hanke.allu.CustomerType
+import fi.hel.haitaton.hanke.application.ApplicationNotFoundException
+import fi.hel.haitaton.hanke.application.ApplicationRepository
+import fi.hel.haitaton.hanke.attachment.common.MockFileClientExtension
+import fi.hel.haitaton.hanke.factory.HakemusFactory
+import fi.hel.haitaton.hanke.factory.HankeFactory
+import fi.hel.haitaton.hanke.permissions.Kayttooikeustaso
+import io.mockk.checkUnnecessaryStub
+import io.mockk.clearAllMocks
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.security.test.context.support.WithMockUser
+import org.springframework.test.context.ActiveProfiles
+
+private const val USERNAME = "test7358"
+
+@SpringBootTest
+@ActiveProfiles("test")
+@WithMockUser(USERNAME)
+@ExtendWith(MockFileClientExtension::class)
+class HakemusServiceITest : DatabaseTest() {
+
+    @Autowired private lateinit var hakemusService: HakemusService
+
+    @Autowired private lateinit var applicationRepository: ApplicationRepository
+
+    @Autowired private lateinit var hakemusFactory: HakemusFactory
+
+    @Autowired private lateinit var hankeFactory: HankeFactory
+
+    @BeforeEach
+    fun clearMocks() {
+        clearAllMocks()
+    }
+
+    @AfterEach
+    fun checkMocks() {
+        checkUnnecessaryStub()
+    }
```

Comment:
There are no mocks in this test class?

Reply:
Ah, copy-paste mistake. Thank you for noticing!